Formula/telnet.rb

```diff
@@ -4,6 +4,8 @@ class Telnet < Formula
   url "https://opensource.apple.com/tarballs/remote_cmds/remote_cmds-54.50.1.tar.gz"
   sha256 "156ddec946c81af1cbbad5cc6e601135245f7300d134a239cda45ff5efd75930"
 
+  conflicts_with "inetutils", because: "both install `telnet' binaries"
```

Comment:
`=>` remains the preferred syntax in core.

Reply:
@DomT4 Apologies, I couldn't get `brew audit` to run properly.